CMakeLists.txt

```diff
@@ -303,6 +303,38 @@ include(CheckTypeSize)
 check_include_file(inttypes.h HAVE_INTTYPES_H)
 check_include_file(stdint.h HAVE_STDINT_H)
 check_include_file(unistd.h HAVE_UNISTD_H)
+check_include_file(sys/capability.h HAVE_SYS_CAPABILITY_H)
+if(HAVE_SYS_CAPABILITY_H)
+  add_definitions(-DHAVE_SYS_CAPABILITY=1)
```

Comment:
Shouldn't this be true only if HAVE_SYS_CAPABILITY (set below on lines 326-336 is true?